packages/near-membrane-base/src/environment.ts

```diff
@@ -61,10 +61,11 @@ export class VirtualEnvironment {
         // prettier-ignore
         const {
             blueConnector,
+            redConnector,
             distortionCallback,
             instrumentation,
             liveTargetCallback,
-            redConnector,
+            revokedProxyCallback,
```

Comment:
This is obviously just a nitpick, but it's something I struggle with: strict alphabetical order, or conceptual grouping? No change requested, just thinking out loud